_projects/2024/100535296/100535296.Rmd

```diff
@@ -0,0 +1,534 @@
+---
+title: "Age Distribution of Athletes by Sport since 1896"
+description: |
+  This project focuses on recreating a The Washington Post graph showing the olympic athlete distribution by sport and age since the inception of the modern games in 1896. 
+categories: "2024"
+author: "Irene Bosque Gala"
+date: "`r Sys.Date()`"
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+---
+
+# ORIGINAL GRAPH
```

Comment:
Please, set all titles in sentence case.

Also, the main title of the post is a level-1 heading. You should start with `##` for sections, then `###` for subsections and so on.